core/domain/voiceover_services_test.py

```diff
@@ -34,51 +34,57 @@
     models.Names.VOICEOVER])
 
 
-class EntityVoiceoverServicesTests(test_utils.GenericTestBase):
-    def test_get_entity_voiceover_returns_correctly(self) -> None:
+class EntityVoiceoversServicesTests(test_utils.GenericTestBase):
+    def test_get_entity_voiceovers_returns_correctly(self) -> None:
         dummy_manual_voiceover_dict: state_domain.VoiceoverDict = {
             'filename': 'filename1.mp3',
             'file_size_bytes': 3000,
             'needs_update': False,
             'duration_secs': 6.1
-            }
+        }
         dummy_autogenerated_voiceover_dict: state_domain.VoiceoverDict = {
             'filename': 'filename2.mp3',
             'file_size_bytes': 3500,
             'needs_update': False,
             'duration_secs': 5.9
         }
         voiceover_models.EntityVoiceoversModel.create_new(
-                feconf.ENTITY_TYPE_EXPLORATION, 'exp_id', 1, 'en-US', {
-                    'content_0': {
-                        'manual': dummy_manual_voiceover_dict,
-                        'auto': dummy_autogenerated_voiceover_dict
-                        }
+            feconf.ENTITY_TYPE_EXPLORATION, 'exp_id', 1, 'en-US', {
+                'content_0': {
+                    feconf.VoiceoverType.MANUAL.value: (
+                        dummy_manual_voiceover_dict),
+                    feconf.VoiceoverType.AUTO.value: (
+                        dummy_autogenerated_voiceover_dict)
                     }
```

Comment:
Deindent this and the next line by 4. They should match their respective opening brace scopes.

Reply:
Done